src/dtclient/connection_info.go

```diff
@@ -6,6 +6,13 @@ import (
 	"github.com/pkg/errors"
```

Comment:
file should be renamed to `activegate_connection_info.go` if it only has activegate relevant functionality